ios/Capacitor/Capacitor/Plugins/CapacitorUrlRequest.swift

```diff
@@ -108,11 +108,11 @@ public class CapacitorUrlRequest: NSObject, URLSessionTaskDelegate {
         }
     }
 
-    public func setRequestHeaders(_ headers: [String: String]) {
+    public func setRequestHeaders(_ headers: [String: Any]) {
```

Comment:
Since `CapacitorUrlRequest` is a public class and `setRequestHeaders(_ headers: [String: String])` is a public method we can't change it to Any now or it would be breaking.
We should keep the existing `setRequestHeaders(_ headers: [String: String])` and add a new `func setRequestHeaders(_ headers: [String: Any])` and probably deprecate the old one.